src/assets/translations/en/main.json

```diff
@@ -926,9 +926,9 @@
         "negativeRatio": "Insufficient sell amount"
       },
       "rateExpired": {
-        "title": "Rate expired",
+        "title": "New Rate",
         "body": "Your previous swap rate has expired. We've fetched a new rate and updated your quote amounts. Please review the latest details before confirming your swap.",
-        "cta": "Ok, got it"
+        "cta": "Accept"
```

Comment:
"Accept" is a bit strange. What if the new rate is 30% worse, and I don't want to "Accept" it?

Even though clicking that button won't actually mean the user is committed, the user doesn't know that and it sounds pretty scary. I'd probably force close the tab if I saw this for a rate I didn't want.

We might want to add a "Back" or something if we are going to have "Accept".

CC @shapeshift/product @twblack88 @reallybeard

Reply:
Totally aligned on a back button of sorts for users to be taken back to input as they might not understand what "Accept" entails I.e they may think it will trigger a sign and broadcast! @reallybeard thoughts?

Reply:
Even the `before confirming your swap` could be scary, does it mean that `Accept` is confirming the swap?

Reply:
@0xApotheosis @NeOMakinG reverted the button to the previous "Ok, got it" in 3dba98942f to merge this in the sake of progression, since the goal is to get this in first and revisit later, this way we keep the exact same CTA/copies as before (except for "Rate Expired" now being "New Rate") and the only actual addition here is the rate changed info

Reply:
I'm fine with it, as long sa we don't write "Accept" in the button